pegasus/emails/hoc_signup_2022_receipt_en.md

```diff
@@ -1,5 +1,6 @@
 ---
 from: "Hadi Partovi (Code.org) <hadi_partovi@code.org>"
+reply-to: hadi_partovi@code.org
```

Comment:
Shouldn't we use `noreply@code.org` here?

Reply:
Emails sent to hadi_partovi@code.org get routed to Support, which can be convenient for people to be able to get help.

Reply:
Maybe we can try this in two phases? First check whether specifying a reply-to improves delivery? And then switch to noreply?

Reply:
The two-phase approach SGTM!